bedrock/base/templates/macros.html

```diff
@@ -349,7 +349,9 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
               <input id="id-input" name="phone-or-email" type="text" required>
             </div>
             <div class="form-submit">
-              <button type="submit" class="button-flat" data-download-os="Android">{{ _('Send') }}</button>
+              <button type="submit" class="button-flat"
+              {% if platform == 'android' %} data-download-os="Android"
+              {% elif platform == 'ios' %} data-download-os="iOS" {% endif %}>{{ _('Send') }}</button>
```

Comment:
To do this we would likely need to use JS to populate the data attribute value when someone makes a selection in the UI, however if you're OK not tracking this for now then we can just add this later if/when we need.